handbook/filesystem.md

```diff
@@ -0,0 +1,80 @@
+## EasyEngine on the Filesystem
+
+All files created by EasyEngine are stored at `/opt/easyengine`
+
+## Sites
+
+By default, all EasyEngine sites are created at `/opt/easyengine/sites`. Configuration and logs of individual site's NGINX, PHP, Postfix are stored in `/opt/easyengine/sites/example.com`. 
```

Comment:
> default

What is a way to override default?

Reply:
> Configuration and logs of

are not source code, database and everything relates to a site is stored under `/opt/easyengine/sites/example.com` 🤔

Reply:
Yes it is.

Reply:
> What is a way to override default?

Earlier there was a provision to override it. I was under the impression it was still there. Looks like it was removed.